spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -32,6 +32,19 @@
         expect(xml.at("Exemptions DomesticPartnerRegular")).to eq(nil)
       end
 
+      it "populates line 8 XML fields" do
+        expect(xml.at("Exemptions YouBlindOrDisabled")).to eq(nil)
+        expect(xml.at("Exemptions SpouseCuPartnerBlindOrDisabled")).to eq(nil)
```

Comment:
[pebble] this sounds like a nitpick but I do think test clarity is important so pebble it is (sorry). I think "populates fields" doesn't match with the behavior of expecting these fields to be nil and not exist, can we update test name to match behavior?